app-backend/api/src/main/scala/user/Routes.scala

```diff
@@ -309,18 +308,6 @@ trait UserRoutes
                     )
 
                   }).transact(xa).unsafeToFuture
-                  _ <- newUser traverse { u =>
```

Comment:
What's the rationale for not needing this? Is it because the anonymized users are all in a specific connection?

Reply:
@jisantuc in the function call above this one, we create user group roles for these new users, so when they log in, they should be able to authenticate correctly, which means that the platform id and the organization id in app metadata will not be used for that. We do not read app metadata on frontend either.